### PR TITLE
additional exports and wrappers

### PR DESCRIPTION
**Pull request by SergejJurecko**

### CODE REVIEW

File: security-framework/src/trust.rs [hunk 1]
```diff
@@ -103,6 +104,11 @@ impl SecTrust {
         unsafe { cvt(SecTrustSetPolicies(self.0, policy.as_CFTypeRef())) }
     }
 
+    /// Returns the public key for a leaf certificate after it has been evaluated.
+    pub fn copy_public_key(&mut self) -> Result<SecKey> {
+        unsafe { Ok(SecKey::wrap_under_get_rule(SecTrustCopyPublicKey(self.0))) }
```
Comment (kornelski) [inline security-framework/src/trust.rs]: Are you sure about the get rule? I think `Copy` functions behave like `Create` ones.

Comment (SergejJurecko) [reply security-framework/src/trust.rs]: Honestly I do not know. We can remove this for now perhaps and leave the low level APIs?

Comment (kornelski) [reply security-framework/src/trust.rs]: It's fine. Just change it to create rule.

Comment (SergejJurecko) [reply security-framework/src/trust.rs]: Done

**Comment by kornelski** [comment]
Thank you

**Pull request merged**